connection-api/src/main/java/org/terracotta/connection/ConnectionFactory.java

```diff
@@ -56,4 +71,17 @@ private static void validateURI(URI uri) throws ConnectionException {
       throw new ConnectionException(e);
     }
   }
+
+  private static Connection findConnectionService(List<String> servers, Properties properties)
+      throws ConnectionException {
+    String scheme = properties.getProperty(ConnectionPropertyNames.CONNECTION_SCHEME, "terracotta");
```

Comment:
I don't understand how it would work for management, which needs to use different schemes to connect to a cluster. We are using: `terracotta://`, `diagnostic://` and `stripe://` (see downstream projects).

Reply:
@anthonydahanne @GaryWKeim : FYI

Reply:
UPDATE: Oh I see! We have to pass the scheme as a property.... @anthonydahanne : this will be an important change for us to keep in mind!